metric_name_builder_bench_test.go

```diff
@@ -0,0 +1,120 @@
+package otlptranslator
+
+import (
+	"fmt"
+	"testing"
+
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+func BenchmarkBuildCompliantMetricName(b *testing.B) {
+	metrics := createTestScenarios()
```

Comment:
`metrics` is a strange name for a collection of scenarios:
```suggestion
	scenarios := createTestScenarios()
```